nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/LogTailer.java

```diff
@@ -82,6 +82,22 @@
      */
     void seek(LogOffset offset);
 
+    /**
+     * Set the current position for this group.
+     * The position is the earliest offset whose timestamp is greater than or equal to the given timestamp
```

Comment:
`Set` -> `Sets` (3rd person declarative rather than 2nd person imperative). Add final period. Please format the Javadoc (you may use `<p>` if needed)

Reply:
The 4 existing method javadocs use "Set"

Reply:
Hah, it means @bdelbosc doesn't follow Javadoc conventions either :)

Reply:
It sets the **committed** positions for assigned partitions of the tailer group to a position corresponding to a timestamp.
After reflexion seek may not be the right method name may me mimic the kafka api with a 
LogOffset offsetForTimestamp(LogPartition partition, long timestamp);
will be easier to understand for instance to do what we want we combine tailer operations: offsetForTimestamp + seek(offset) + commit

Reply:
also gives some hint on what is the timestamp mean, for kafka it is by default the LogAppendTime (the time at which the broker receive the record) but it could be set to CreateTime